Mathlib/Topology/Constructions.lean

```diff
@@ -1100,8 +1100,8 @@ theorem closure_subtype {x : { a // p a }} {s : Set { a // p a }} :
 #align closure_subtype closure_subtype
 
 theorem continuousAt_codRestrict_iff {f : α → β} {t : Set β} (h1 : ∀ x, f x ∈ t) {x : α} :
```

Comment:
I recognise you're just golfing the proof but while you're here how about adding `simp`
```suggestion
@[simp]
theorem continuousAt_codRestrict_iff {f : α → β} {t : Set β} (h1 : ∀ x, f x ∈ t) {x : α} :
```